### PR TITLE
Add jupyter_cms recipe

### PR DESCRIPTION
**Pull request by parente**
Starting with version 0.6.1 from https://github.com/jupyter-incubator/contentmanagement

### CODE REVIEW

**Comment by parente** [comment]
Looks like the circleci build finished when I visit it, but the indicator is stuck on github.

**Comment by jakirkham** [comment]
Yeah, the GitHub API had some issues yesterday. Some of the status indicators were negatively effected as a consequence. In the past, restarting the CI has helped. I have done so with CircleCI here. Let's see if that fixes it.

**Comment by jakirkham** [comment]
Seems to have done the trick.

File: recipes/jupyter_cms/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - jupyter-cms = jupyter_cms.extensionapp:main
+    - jupyter-bundler = jupyter_cms.bundlerapp:main
+
+
```
Comment (jakirkham) [inline recipes/jupyter_cms/meta.yaml]: nit: one newline please.

File: recipes/jupyter_cms/post-link.bat [hunk 2]
```diff
@@ -0,0 +1,3 @@
+@echo off
+"%PREFIX%\Scripts\jupyter-cms" quick-setup --sys-prefix > "%PREFIX%\.messages.txt" 2>&1
+
```
Comment (jakirkham) [inline recipes/jupyter_cms/post-link.bat]: nit: please keep only the terminal newline.

File: recipes/jupyter_cms/pre-unlink.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+"$PREFIX/bin/jupyter-cms" quick-remove --sys-prefix > $PREFIX/.messages.txt 2>&1
+
```
Comment (jakirkham) [inline recipes/jupyter_cms/pre-unlink.sh]: nit: please keep only the terminal newline.

File: recipes/jupyter_cms/meta.yaml (hunk 1 above)
Comment (parente) [inline recipes/jupyter_cms/meta.yaml]: Fixed.

File: recipes/jupyter_cms/post-link.bat (hunk 2 above)
Comment (parente) [inline recipes/jupyter_cms/post-link.bat]: Fixed.

File: recipes/jupyter_cms/pre-unlink.sh (hunk 3 above)
Comment (parente) [inline recipes/jupyter_cms/pre-unlink.sh]: Fixed.

**Comment by parente** [comment]
@jakirkham should be good to go

**Comment by jakirkham** [comment]
Thanks @parente.

**Pull request merged**